dao/makefile

```diff
@@ -19,10 +19,6 @@ build:
 	cd elasticsearch && make build
 	go build
 
-test: build
-	cd elasticsearch && make test
-	go test $(GOTEST_FLAGS)
-
 clean:
```

Comment:
Remove recursive make testing scheme in favor of go test ./... recursive testing scheme.